build_web_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -114,56 +115,33 @@ Future<void> _createDevCompilerModule(Module module, BuildStep buildStep,
       '-o',
       jsOutputFile.path,
       debugMode ? '--source-map' : '--no-source-map',
+      for (var dep in transitiveDeps) _summaryArg(dep),
```

Comment:
:champagne: cc @munificent this might be one of the best ones I have seen yet...

Reply:
Yeah, it works so well for command line args that it almost seems tailor-made for them. :)